src/renderer/constants.js

```diff
@@ -7,10 +7,13 @@ const messages = {
   de,
 };
 
-const langs = [
-  'en',
-  'de',
-];
+const languages = [{
+  label: 'English',
+  value: 'en',
+}, {
+  label: 'German',
```

Comment:
Oh we need to comment out being able to select German in here until we have all german strings. can you just leave it where english is the only thing in the list for now.

Reply:
Requesting change for this